Sources/Segment/Utilities/QueueTimer.swift

```diff
@@ -73,18 +73,18 @@ internal class QueueTimer {
 
 extension TimeInterval {
     static func milliseconds(_ value: Int) -> TimeInterval {
-        return TimeInterval(value / 1000)
+        return TimeInterval(value) / 1000
```

Comment:
This calculation had an integer-division problem. My change makes it so that we're dividing `Double` types rather than `Int` types, and therefore are no longer truncating.